system-parachains/bridge-hubs/bridge-hub-kusama/primitives/src/lib.rs

```diff
@@ -19,6 +19,9 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+extern crate alloc;
+
+use alloc as sp_std;
```

Comment:
a bit surprising rename here, maybe better to replace usage with alloc no ?

Reply:
This is because a macro expanded as `sp_std:: something` (can't remember what thing right now). And since you can't change the macro, you can at least help it a bit. So that's what I did here.

Reply:
At the end we reintroduced `sp-std` until next patch.